TASK00391598, TASK00391887, TASK00392637: add multiple sites

jbickar opened the pull request.

jbickar committed:
TASK00391598, TASK00391887, TASK00392637: add multiple sites

jbickar committed:
fixup! WIP

mdyoung3 committed:
Adding commerce, orm and oed sites to nightly script.

mdyoung3 committed:
Adding osed9

mdyoung3 committed:
Adding ora.stanford.edu
improvement.stanford.edu
bahr.stanford.edu

mdyoung3 committed:
Removed idea configs.

mdyoung3 committed:
adding beamd9

mdyoung3 committed:
Removing idea configs

mdyoung3 committed:
Adding phpstorm config folder to gitignore

jbickar committed:
TASK00400294: acronyms

mdyoung3 committed:
Adding community and campusengagement

jbickar committed:
fixup! WIP

jbickar committed:
fixup! WIP

mdyoung3 commented:
https://communitycenters.stanford.edu/  (19631)
https://fsl.stanford.edu/ (23256)
shenlab.sites.stanford.edu (26276)
ose.stanford.edu (23151)
communitycenters.stanford.edu (19631)

mdyoung3 commented:
@jbickar I took a look at those service now tickets. Can you confirm if those are supposed to be on this PR. I only see the site id for ose.stanford.edu in the script: https://github.com/SU-SWS/acsf_setup/blob/92c61f0165e0a9513958cd667e80f4cdb7dc75fe/stage-sites.sh

jbickar committed:
fixup! WIP

jbickar commented:
Removed `17676`, which was causing the script to fail.

Commit:
Added  TASK00410441

jbickar committed:
TASK00413563: add computerforumd9

Commit:
TASK00415099

mdyoung3 reviewed (approved):
Looks good to me.

mdyoung3 commented:
@jbickar I can take care of the conflict.

Commit:
Adding Harzenberglab

Commit:
Added MPE

Commit:
TASK00418945

Commit:
TASK00421039

imonroe committed:
added cifed8 (18746)

Commit:
TASK00429591

Commit:
TASK00433578

Commit:
TASK00434864

mdyoung3 committed:
Added the following:
--https://compression.stanford.edu/
--https://facultydevelopment.stanford.edu/
--https://ogc.stanford.edu/
--https://equity.stanford.edu/
--https://glo.stanford.edu/
--https://gradadmissions.stanford.edu/
--https://biomechatronics.stanford.edu/

mdyoung3 committed:
TASK00446760

mdyoung3 committed:
TASK00450330

mdyoung3 committed:
TASK00452970

mdyoung3 committed:
TASK00454149

jbickar committed:
fixup! WIP

mdyoung3 committed:
TASK00465193
